jsapp/js/components/formActivity.es6

```diff
@@ -0,0 +1,7 @@
+export default function FormActivity() {
```

Comment:
Why not `formActivity.tsx`? Also I think it would be a good idea to make a separate directory for that file, e.g. `activity/formActivity.tsx`. I imagine we would need some styles file for it in near future.

`jsapp/js/components` directory was historically a bottomless pit for all js files, and we still have quite a lot of mess in there. So take it into account when trying to find stuff to mimic :D